pythermalcomfort/models/two_nodes_gagge_ji.py

```
@@ -0,0 +1,455 @@
+        alfa = 0.0417737 + 0.7451832 / (m_bl + 0.5854417)
+
+        # Sweat rate (SWR) - accounting for the metabolic rate
+        m_rsw = (
+            c_swe
+            * c_sw
+            * ((1 - alfa) * t_cr_sw + (alfa + a_cof) * t_sk_sw)
+            * math.exp(t_sk_sw / 10.7)
+        )
+        # Apply max sweating rate
+        if m_rsw > m_rsw_max:
+            m_rsw = m_rsw_max
```

Pylint (3.3.7)

[refactor] Consider using 'm_rsw = min(m_rsw, m_rsw_max)' instead of unnecessary if block

(R1730)